src/test/java/com/github/bibsysdev/nva/cristin/person/CristinPersonsHandlerTest.java

```diff
@@ -0,0 +1,227 @@
+package com.github.bibsysdev.nva.cristin.person;
+
+import static com.github.bibsysdev.nva.cristin.person.CristinApiClient.PERSON_LOOKUP_CONTEXT_URL;
+import static com.github.bibsysdev.nva.cristin.person.CristinHandler.LANGUAGE_INVALID_ERROR_MESSAGE;
+import static com.github.bibsysdev.nva.cristin.person.CristinPersonsHandler.LANGUAGE_QUERY_PARAMETER;
+import static com.github.bibsysdev.nva.cristin.person.CristinPersonsHandler.NAME_QUERY_PARAMETER;
+import static nva.commons.apigateway.ApiGatewayHandler.APPLICATION_PROBLEM_JSON;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.bibsysdev.nva.cristin.person.model.cristin.CristinPerson;
+import com.github.bibsysdev.nva.cristin.person.model.nva.NvaPerson;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.core.Environment;
+import nva.commons.core.JsonUtils;
+import nva.commons.core.ioutils.IoUtils;
+import org.apache.commons.codec.Charsets;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.mockito.Mockito;
+
+public class CristinPersonsHandlerTest {
+
+    private static final String LANGUAGE_KEY = "language";
```

Comment:
I think the tests here should be more of the format 
```
itReturnsTwoEnrichedPersonswhenInputContainsNameAndLangAndNameMatchesTwoPersonsInCristin,
itRetruns1EnrichedAnd1NotEnrichedPersonWhenInputContainsNameAndLangAndOneQueryFails
itReturnsBadRequestWhenInputDoesNotContainName
...
```

Reply:
I will create Jira task for this further work. I believe it is out of scope for the PR as we'd have to do the same for the nva-cristin-projects project and there is large degree of similarity in the test coverage of these projects.